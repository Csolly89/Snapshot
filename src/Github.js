import Card from 'react-bootstrap/Card';
import FamilyImg from './Family.jpg'

function Github () {
    return (
        <div>
            <Card style={{ width: '18rem' }}>
                <Card.Img variant="top" src={FamilyImg} />
                <Card.Body>
                    <Card.Title>For Family</Card.Title>
                    <Card.Text>
                        Nothings impossible when you have Family - Dom
                    </Card.Text>
                </Card.Body>
            </Card>
        </div>
    )
}

export default Github