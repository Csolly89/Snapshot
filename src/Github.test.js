import Github from './Github';
import renderer from 'react-test-renderer'


test('renders a snapshot', () => {
    const tree = renderer.create(<Github/>).toJSON()
    expect(tree).toMatchSnapshot()
})