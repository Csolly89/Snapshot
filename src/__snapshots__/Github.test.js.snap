// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="Family.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        For Family
      </div>
      <p
        className="card-text"
      >
        Nothings impossible when you have Family - Dom
      </p>
    </div>
  </div>
</div>
`;
